src/lib/types/stores/client-instance-store.ts

```diff
@@ -29,6 +29,9 @@ export interface IClientInstanceStore
     groupApplicationsBySdk(): Promise<
         { sdkVersion: string; applications: string[] }[]
     >;
+    groupApplicationsBySdkAndProject(
```

Comment:
are we grouping by project? I thought we're only filtering/where by project? Maybe we can add optional projectId to the method signature and only keep 1 method?